src/services/Content/ContentApi.ts

```diff
@@ -330,12 +330,30 @@ export const introspectRepository: (
   return data;
 };
 
+export const triggerSnapshot: (repositoryUUID: string) => Promise<void> = async (
+  repositoryUUID,
+) => {
+  try {
+    const { data } = await axios.post(
+      `/api/content_sources/v1.0/repositories/${repositoryUUID}/snapshot/`,
```

Comment:
oh and this should be 'content-sources' not 'content_sources', apologies for not catching that earlier.